src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -203,6 +248,30 @@ private static String getTextOrEmptyString(TextField textField) {
     return textField.getText() != null ? textField.getText().trim() : "";
   }
 
+  private static String convertConfigWinnerElectionModeToGuiText(ContestConfig config) {
+    switch (config.getWinnerElectionMode().toString()) {
+      case "standard" -> {
+        return config.getNumberOfWinners() > 1 ? "Multi-winner allow multiple winners per round"
+            : "Single-winner majority determines winner";
+      }
+      case "singleSeatContinueUntilTwoCandidatesRemain" -> {
+        return "Single-winner majority determines winner";
+      }
+      case "multiSeatAllowOnlyOneWinnerPerRound" -> {
+        return "Multi-winner allow only one winner per round";
+      }
+      case "multiSeatBottomsUp" -> {
+        return config.getNumberOfWinners() == 0
+            || config.getMultiSeatBottomsUpPercentageThreshold() != null
```

Comment:
if getMultiSeatBottomsUpPercentageThreshold() is not null and getNumberOfWinners is non-zero, something is wrong

Reply:
Indeed, keep in mind that this logic is for loading a config file, which could indeed be configured incorrectly. Subsequent validation would catch it if so. This logic is just to best decide what should appear in the Winner Election Mode text box based off what it loads. Given that, do you agree that this handling makes sense?